CHANGELOG.md

```diff
@@ -32,6 +32,7 @@ of videos.
   default. If the browser sends an incorrect packet rate, this will be logged
   as an error in the console.
 - Add a warning log in `InMemoryJSONEventBuffer`'s `send` function when retrying starts.
+- Added support for Chromium WebView
```

Comment:
I think we can say Android WebView here, even though the user agent is chromium webview

Reply:
Changed!